core/arch/arm/kernel/spmc_sp_handler.c

```diff
@@ -554,12 +555,14 @@ static void ffa_mem_retrieve(struct thread_smc_args *args,
 						       address_offset);
 
 		va = READ_ONCE(mem_region->address_range_array[0].address);
+		ret_map = sp_map_shared(caller_sp, receiver, smem,  &va);
 
-		ret = sp_map_shared(caller_sp, receiver, smem,  &va);
-		if (ret) {
+		if (ret_map) {
+			EMSG("Could not map memory region (%x)", ret_map);
```

Comment:
filts in 1 line: `EMSG("Could not map memory region; %#"PRIx32, ret_map);`

Reply:
Done